### PR TITLE
New default instance icons

### PR DESCRIPTION
**Pull request by santiagocezar**
A bit more in brand with the new logo, blocks are not final

edit: added gradients and some of the blocks
image

### CODE REVIEW

**Comment by RaptaG** [comment]
The path should be renamed from `/…/multimc/…:` to `/…/prism/…/`

**Comment by Soup-64** [comment]
Looks good but I would definitely stylize the curseforge logo a bit more since the other logos are stylized as well, mainly with the fire and rounding the anvil a bit more, also might want to use the same red that's present in the bricks, and TNT for color pallete consistency.

**Comment by DioEgizio** [comment]
> The path should be renamed from `/…/multimc/…:` to `/…/prism/…/`

That would break many stuff, and it's multimc because it's in the same folder of the legacy iconpack

**Comment by Madis0** [comment]
Is there a reason these icons must be based on the old ones? I would suggest choosing entirely new icons, like #190.

**Comment by BeansBeefBroccoli** [comment]
I suggest icons for mod loaders too

**Comment by santiagocezar** [comment]
> Is there a reason these icons must be based on the old ones? I would suggest choosing entirely new icons, like #190.

essentially, to replace the old ones without leaving instances with no icons

**Comment by DioEgizio** [comment]
Any update on this?

**Comment by santiagocezar** [comment]
Oh yeah, I thought these were superseded by the ones by Fish, but yeah, I'll should probably just finish them.

Does anyone know some tool to convert these to SVG Tiny? I looked up and all I found was Adobe Illustrator...

**Comment by Scrumplex** [comment]
As long as you are just using normal paths, groups and shapes it should be fine. The only notable feature missing in SVG tiny are masks. Best way to test is to just add your SVGs to Prism.

Edit: so you don't really need to export your SVGs in a specific format or anything

**Comment by santiagocezar** [comment]
imagen

Yep, I was using masks quite a bit but that's fixed now! Though these look kinda blurry, and the backgrounds are bleeding through the borders of the overlapping shapes which uh, yeah it looks bad.

This is how it looks on Inkscape at 2x

imagen

**Comment by AshtakaOOf** [comment]
Some icons should have their names changed tbh:
flame to curseforge, ftb_logo to ftb, squarecreeper to creepy, prismlauncher to prism.
Just a suggestion. :)

**Comment by DioEgizio** [comment]
flame is the internal name of curseforge on prism, and pretty sure there are 2 ftb logos

**Comment by santiagocezar** [comment]
also to keep compatibility with instances already using these icons (tho i didn't make an icon for ftb_glow, it seems kinda redundant)

**Comment by AshtakaOOf** [comment]
Shouldn't this PR put the old icons somewhere else, so peoples who like the old one can still use them?

**Comment by Scrumplex** [comment]
Ideally, we should just keep the old icons around. I heard that some people like them :eyes:

**Comment by santiagocezar** [comment]
_In the end, the two nations lived together in peace._ (hopefully)

Captura desde 2022-11-22 13-02-29

**Comment by TheKodeToad** [comment]
This is making me wonder why all of the name are with underscores. As long as the enderpearl is still there :cry:.

**Comment by santiagocezar** [comment]
Icons are now pixel aligned!

imagen

**Comment by DioEgizio** [comment]
fix conflicts and DCO please

**Review by DioEgizio — APPROVED** [review]

File: launcher/resources/multimc/index.theme [hunk 1]
```diff
@@ -2,7 +2,7 @@
 Name=multimc
 Comment=Default Icons
 Inherits=default
-Directories=8x8,16x16,22x22,24x24,32x32,32x32/instances,48x48,50x50/instances,64x64,128x128/instances,256x256,scalable,scalable/instances
```
Comment (Scrumplex) [inline launcher/resources/multimc/index.theme]: any reason why you removed these?

File: launcher/resources/multimc/multimc.qrc [hunk 2]
```diff
@@ -7,7 +7,7 @@
         <file>scalable/reddit-alien.svg</file>
 
         <!-- Icon for CurseForge. CC0 -->
-        <file alias="128x128/flame.png">128x128/instances/flame.png</file>
+        <file alias="128x128/flame.png">scalable/instances/flame.svg</file>
```
Comment (Scrumplex) [inline launcher/resources/multimc/multimc.qrc]: probably CC-BY-SA 4.0, Santiago Cézar?

File: launcher/resources/multimc/multimc.qrc [hunk 3]
```diff
@@ -254,65 +254,92 @@
         <!-- discord logo icon thing. from discord. traced from bitmap -->
         <file>scalable/discord.svg</file>
 
+		<!-- flat instance icons CC BY-SA 4.0, Santiago Cézar -->
```
Comment (Scrumplex) [inline launcher/resources/multimc/multimc.qrc]: tabs detected!!

File: launcher/resources/multimc/multimc.qrc [hunk 4]
```diff
@@ -254,65 +254,92 @@
...
+		
         <!-- instance icons -->
-        <file>32x32/instances/chicken.png</file>
-        <file>128x128/instances/chicken.png</file>
-
-        <file>32x32/instances/creeper.png</file>
-        <file>128x128/instances/creeper.png</file>
+        <file>32x32/instances/chicken_legacy.png</file>
+        <file>128x128/instances/chicken_legacy.png</file>
+
```
Comment (Scrumplex) [inline launcher/resources/multimc/multimc.qrc]: extra spaces detected!!

**Review by Scrumplex — COMMENTED** [review]
Looks good overall. Just these small changes :D

**Comment by BeansBeefBroccoli** [comment]
Herobrine Leg

**Comment by DioEgizio** [comment]
conflicting with develop

**Review by DioEgizio — APPROVED** [review]

**Review by Scrumplex — APPROVED** [review]
Thanks a lot!

**Review by flowln — APPROVED** [review]
super cool, thanks!

**Pull request merged**